timeline-chart/src/components/time-graph-axis-scale.ts

```diff
@@ -91,7 +90,7 @@ export class TimeGraphAxisScale extends TimeGraphComponent<null> {
                     };
                     let label;
                     if (drawLabels && this.unitController.numberTranslator) {
-                        label = this.unitController.numberTranslator(time);
+                        label = this.unitController.numberTranslator(time - this.unitController.offset);
```

Comment:
in the numberTranslator of the theia-trace-extension the offset is subtracted from the value passed. I haven't run the code, but it doesn't feel right to subtract it here too.

Reply:
In trace-context-component.tsx the start time (offset) is added to the provided number.

Reply:
I see, the variable `time` here is now the "real" time and not the offset time as it was in the previous code. Makes sense.